zanata-war/src/main/webapp/WEB-INF/pages.xml

```diff
@@ -136,13 +131,12 @@
     <action execute="#{activate.activate}" />
   </page>
 
-  <page view-id="/account/email_validation.xhtml" login-required="true">
+  <page view-id="/account/email_validation.xhtml">
```

Comment:
Oh, I see there's a check in a Create method.  

- [ ] Just to be doubly safe, I wouldn't mind seeing the annotations too.  Perhaps we could intercept bean creation in CDI with an extension.